assets/js/partners/contact.js

```diff
@@ -0,0 +1,129 @@
+window.clearCaptchaError = () => {
+  const error = document.getElementById('captcha-error-message');
+  if (error.textContent) {
+    error.textContent = '';
+    error.classList.add('display-none');
+  }
+};
+
+function fetchTextInput(id) {
+  return document.querySelector(`#contact-us-form #${id}`).value;
+}
+
+function fetchSelectInput(name) {
+  return document.querySelector(`#contact-us-form select[name="${name}"]`).value;
+}
+
+async function submitPartnerContact(event) {
+  event.preventDefault();
+
+  const debug = Array.prototype.slice.apply(document.getElementsByName('debug'))[0];
+  const form = document.getElementById('contact-us-form');
```

Comment:
For aggregating all the form data, one handy way to convert the form to an object of name-value pairs is like this:

```ts
const formData = Object.fromEntries(new FormData(form));
```

Might help avoid some of the variable assignment and field value helpers.